buildpacks/jvm/src/openjdk_artifact.rs

```diff
@@ -0,0 +1,170 @@
+use crate::openjdk_version::OpenJdkVersion;
+use inventory::version::{ArtifactRequirement, VersionRequirement};
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub(crate) struct OpenJdkArtifactMetadata {
```

Comment:
(optional) Would `#[serde(deny_unknown_fields)]` make sense here for completeness / easier debugging?